frontend/server/src/main/java/org/pytorch/serve/util/ConnectorType.java

```diff
@@ -3,5 +3,6 @@
 public enum ConnectorType {
     INFERENCE_CONNECTOR,
     MANAGEMENT_CONNECTOR,
+    METRICS_CONNECTOR,
     BOTH
```

Comment:
can we change this to `ALL` and use the same during server initialization?

Reply:
This is used in UTs and a few other classes for getting the different channels for the connectors

Reply:
Can't we use `ALL` instead of `BOTH` and initialize all three channels using this keyword? `BOTH` now looks a bit odd here.

Reply:
Ah, I see what you meant. Will do

Reply:
Done